201901012 updatProject/Presenter/UpdateClientController.cs

```diff
@@ -0,0 +1,263 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.IO;
+using System.Linq;
+using System.Net;
+using System.Text;
+using UpdateUpload.Interface;
+using UpdateUpload.Model;
+using WinSCP;
+
+namespace UpdateUpload.Presenter
+{
+    class UpdateClientController : IUpdateClientController
+    {
+        #region Variable
+
+        #region Constant
+
+        private const string VersionTxtFileName = "Version.txt";
+        private const string RequestUriString = "ftp://192.168.53.26/";
```

Comment:
If there is a way to use 'environmental variables' in c# (and there is), use the variable to store setting values, not in the class.